tests/unit/javascript/stylesheet/test_stylesheet.py

```diff
@@ -0,0 +1,125 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Jay Kamat
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for stylesheet.js."""
+
+import os
+import pytest
+
+from PyQt5.QtWebEngineWidgets import QWebEngineProfile
```

Comment:
This is currently failing without webengine, what's the best way to skip this file if running without webkit installed?

Reply:
Something like:

```python
QtWebEngineWidgets = pytest.importorskip("PyQt5.QtWebEngineWidgets")
QWebEngineProfile = QtWebEngineWidgets.QWebEngineProfile
```

(in the hope that pylint/flake8 will be fine with that :laughing:)